### PR TITLE
copyedit prompt-design vignette: code generation through Token usage

### PR DESCRIPTION
**Pull request by lindbrook**

### CODE REVIEW

File: vignettes/prompt-design.Rmd [hunk 1]
```diff
@@ -76,15 +76,15 @@ I'll use `chat_claude()` for this problem because in our experience it does the
...
+When I don't provide a system prompt, I sometimes get answers in different languages or different styles of R code:
 
 ```{r}
 #| label: code-basic
 chat <- chat_claude()
 chat$chat(question)
 ```
 
-I can ensure that I always get R code in a given style by providing a system prompt:
+To ensure that I always get R code in a given style, I provide a system prompt:
```
Comment (hadley) [inline vignettes/prompt-design.Rmd:87]: The previous seems a bit clearer to me.

File: vignettes/prompt-design.Rmd [hunk 2]
```diff
@@ -226,7 +226,7 @@ chat <- chat_openai(instruct_json)
 chat$chat(ingredients)
 ```
 
-(I don't know if the colour text, "You're an expert baker who also loves JSON", does anything, but I like to think this helps the LLM get into the right mindset of a very nerdy baker.)
+(I don't know if the text colouring, "You're an expert baker who also loves JSON", does anything, but I like to think this helps the LLM get into the right mindset of a very nerdy baker.)
```
Comment (hadley) [inline vignettes/prompt-design.Rmd:229]: I think this changes the meaning, which suggests "colour text" isn't clear. Maybe something like "I don't know if the hint ...` would be better?

File: vignettes/prompt-design.Rmd [hunk 3]
```diff
@@ -251,10 +251,10 @@ chat <- chat_openai(c(instruct_json, instruct_weight))
 chat$chat(ingredients)
 ```
 
-Just providing the examples seems to work remarkably well. But I found it useful to also include description of what the examples are trying to accomplish. I'm not sure if this helps the LLM or not, but it certainly makes it easier for me to understand the organisation of the whole prompt and check that I've covered the key pieces that I'm interested in.
+Just providing the examples seems to work remarkably well. But I found it useful to also include a description of what the examples are trying to accomplish. I'm not sure if this helps the LLM or not, but it certainly makes it easier for me to understand the organisation of the whole prompt and check that I've covered the key pieces I'm interested in.
 
 ```{r}
-#| cacahe: false
+#| cache: false
```
Comment (hadley) [inline vignettes/prompt-design.Rmd:257]: Ooops

**Review by hadley — COMMENTED** [review]
Thanks for your continued help with this. It definitely makes a big difference to the quality of the final product!

File: vignettes/prompt-design.Rmd (hunk 2 above)
Comment (lindbrook) [reply vignettes/prompt-design.Rmd:229]: (I don't know if the hint or colour text, "You're an expert baker who also loves JSON", actually does anything, but I like to think it helps the LLM get into the mindset of a very nerdy baker.)

File: vignettes/prompt-design.Rmd (hunk 3 above)
Comment (lindbrook) [reply vignettes/prompt-design.Rmd:257]: That reminds me. Up on Line 40 setup chunk, I added an "echo: false" to hide what seemed to me to be a single stray line of code: 

"library(elmer)"

Should that actually be visible (without  "echo: false")? I think I saw the same some other documents/vignettes.

Comment (hadley) [reply vignettes/prompt-design.Rmd:257]: Yeah, I think it should be visible. In general, we like to make it obvious that we're loading the package at the start of vignettes.

File: vignettes/prompt-design.Rmd (hunk 1 above)
Comment (hadley) [reply vignettes/prompt-design.Rmd:86]: Yeah, I like that.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**